Resources/Prototypes/Recipes/Lathes/sheet.yml

```diff
@@ -7,6 +7,16 @@
   completetime: 2
   materials:
     Steel: 100
+    
+- type: latheRecipe
+  id: SheetSteel5
+  icon:
+    sprite: Objects/Materials/Sheets/metal.rsi
+    state: steel
+  result: SheetSteel5
```

Comment:
Lathes should just be adjusted to be queueable or the likes if you want this rather than entirely new prototypes.